src/main/java/org/cyclonedx/model/Pedigree.java

```diff
@@ -20,43 +20,53 @@
 
 import java.util.List;
 import java.util.Objects;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
 import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlElementWrapper;
 import com.fasterxml.jackson.dataformat.xml.annotation.JacksonXmlProperty;
+import org.cyclonedx.util.ComponentWrapperDeserializer;
 
 @SuppressWarnings("unused")
-@JsonPropertyOrder({"ancestors", "commits", "notes"})
+@JsonPropertyOrder({"ancestors", "descendants", "variants", "commits", "notes"})
 public class Pedigree extends ExtensibleElement {
 
-    private List<Component> ancestors;
-    private List<Component> descendants;
-    private List<Component> variants;
+    @JacksonXmlProperty(localName = "ancestors")
+    @JsonProperty(value = "ancestors")
```

Comment:
I don't think we need this annotation, same for the other `JacksonProperty`

Reply:
Removed them, and tests pass!

Reply:
In: 48035e0